components/psst/common/pref_names.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/psst/common/pref_names.h"
+
+#include <optional>
+
+#include "base/feature_list.h"
+#include "base/strings/string_util.h"
+#include "base/values.h"
+#include "brave/components/psst/common/features.h"
+#include "components/prefs/pref_registry_simple.h"
+#include "components/prefs/pref_service.h"
+#include "components/prefs/scoped_user_pref_update.h"
+
+namespace psst {
+
+void RegisterProfilePrefs(PrefRegistrySimple* registry) {
+  if (base::FeatureList::IsEnabled(psst::features::kEnablePsst)) {
+    registry->RegisterBooleanPref(prefs::kPsstEnabled, true);
+    registry->RegisterDictionaryPref(prefs::kPsstSettingsPref);
+  }
+}
+
+bool IsPsstEnabled(PrefService* prefs) {
```

Comment:
helpers are not necessary for simple pref get/set. These are meant to be used directly and helpers should only be used for more complex checks. If the checks require helpers, the prefs themselves should not be public